lib/ui/view/edit_profile/edit_profile_view.dart

```diff
@@ -19,84 +25,124 @@ class EditProfileView extends StatelessWidget {
           title: Text("Edit Profile"),
           actions: [
             TextButton(
-              onPressed: () {},
+              onPressed: () {
+                model.updateProfile();
+                model.test();
```

Comment:
Why calling another function? it can be called when updateProfile() is done in the viewmodel. You can use an arrow function when you want to make the changes.